### DISCUSSION TITLE
Add mouse gestures feature

### DISCUSSION THREAD

**Discussion by baalaan** [zen-browser/desktop#1069] [Ideas]
Hi, I love the built-in mouse gestures feature that browsers like Edge/Opera/Vivaldi have. Could you please add this feature if it's not too complicated? Thank you
![edge mouse gestures](https://github.com/user-attachments/assets/6b614ee0-ce33-456c-ba4a-a84fbf66742f)

**Comment by giuli635** [comment]
You can try this: https://addons.mozilla.org/en-US/firefox/addon/gesturefy/

**Comment by otowa1** [comment]
I'm also voting for this. The experience is much better with the built-in, especially when open a new tab.

**Reply by tan2meng** [reply]
and close the current one, move to the right tab left tab and more...

**Comment by miltoncandelero** [comment]
I want a gesture to swap worspaces. Like the two finger swap on the tab window but with mouse. 🥲

**Comment by Shh37** [comment]
voted
I'm using gesturefy right now, but I'm not happy with it, such as not being able to open floating url bar.

**Comment by leic4u** [comment]
Gesturefy does not work on restricted pages. Please consider adding built-in mouse gestures to Zen Browser to support mouse gestures on all pages.

**Reply by cmcpasserby** [reply]
Yeah multiple times i have tried to switch over to ff, and could not since gestures were just not reliable. So ended up on Vivaldi but really want to use Zen since it has some very nice concepts and feels a bit snappier overall.
